FreeRTOS/FreeRTOS-Kernel
# Indent contents of a taskENTER_CRITICAL/taskEXIT_CRITICAL block.

Style only:  Indent and bracket the contents of a taskENTER_CRITICAL/taskEXIT_CRITICAL block.


Test Steps
-----------
No functional changes.

Related Issue
-----------
None.


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
